vimish-fold.el

```diff
@@ -415,6 +451,23 @@ This feature needs `avy' package."
            folds-before-point)))
       (message "No more folds before point"))))
 
+(defun vimish-fold-previous-existing-fold ()
+  "Jump to previous vimish region in current buffer."
```

Comment:
How this is different from `vimish-fold-previous-fold`? It'd help if you mention the differences in the docs string too.

Reply:
Not sure about this one.

The difference is that if you have the following:

```
# {{{ Fold 1
# }}}

# {{{ Fold 2
# {{{ Fold 2.1
# }}}
# }}}
```

When you are on `Fold 2.1`, the old function will make you go back to `Fold 1` because `Fold 2` is in the region of `Fold 2.1`. The new function makes you go back to `Fold 2`.

I thought that a new function was better since it does not alter the behavior but works fine without nested folds. It is however just a little bit slower since we process a little bit the list of folds (vimish-fold--folds-beginning-in).

Done, tell me if you prefer another solution for this :)